src/core/Form/Textarea/Textarea.baseStyles.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { disabledCursor } from '../../../components/utils/css';
+import { element, font } from '../../theme/reset';
+import { focus } from '../../theme/utils/focus';
+
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
+
+    &.fi-textarea {
+      display: flex;
+      flex-direction: column;
+      color: ${theme.colors.blackBase};
+
+      & .fi-textarea_label {
+        ${font({ theme })('actionElementInnerTextBold')};
+        color: ${theme.colors.blackBase};
+      }
+
+      & .fi-textarea_optionalText {
+        ${theme.typography.bodyTextSmall};
+        margin-left: ${theme.spacing.xxs};
+      }
+
+      & .fi-textarea_hintText {
+        display: block;
+        color: ${theme.colors.depthDark1};
+        ${theme.typography.bodyTextSmall};
+      }
+
+      & .fi-textarea_textarea {
+        border-radius: 2px;
+        border: 1px solid ${theme.colors.depthLight1};
+        box-shadow: ${theme.shadows.actionElementBoxShadow};
+        padding: 8px 14px 13px 10px;
+        margin-top: ${theme.spacing.insetL};
+
+        &:focus {
+          ${focus({ theme, noPseudo: true, variant: 'boxShadow' })}
+        }
+        ::placeholder {
+          font-style: italic;
+        }
+      }
+
+      & .fi-textarea_statusText {
```

Comment:
Was it so that there is no token for this text type?

Reply:
I think there was not matching token as also Checkbox does it this way.

Reply:
Yeah, I think the token was left in the air waiting for implementation and styling details from both us and the designers.